### PR TITLE
GUI tweaks

### PR DESCRIPTION
**Pull request by sklencar**
different color for trash in externalResource widget
Change in title for featurePanel
fixed warning for empty string as AttributeValue

related to https://github.com/qgis/QGIS/pull/9388

### CODE REVIEW

File: app/qml/FeaturePanel.qml [hunk 1]
```diff
@@ -175,8 +175,9 @@ Drawer {
                     property color backgroundColorInactive: "grey"
                     property color fontColor: InputStyle.fontColor
                     property color activeColor: InputStyle.fontColor
+                    property color attentionColor: "red"
                     property color normalColor: InputStyle.panelBackgroundLight
-                    property real cornerRadius: 1
+                    property real cornerRadius: 15
```
Comment (wonder-sk) [inline app/qml/FeaturePanel.qml]: This should be specified using dp

File: app/qml/FeaturePanel.qml [hunk 2]
```diff
@@ -175,8 +175,9 @@ Drawer {
                     property color backgroundColorInactive: "grey"
                     property color fontColor: InputStyle.fontColor
                     property color activeColor: InputStyle.fontColor
+                    property color attentionColor: "red"
```
Comment (wonder-sk) [inline app/qml/FeaturePanel.qml]: Can we have a bit darker color so it better matches with the other dark green buttons... e.g. #aa0000 ?

**Comment by sklencar** [comment]
Sample screenshots:

screenshot_2019-03-05-18-25-08-231_uk co lutraconsulting
screenshot_2019-03-05-18-24-54-989_uk co lutraconsulting
screenshot_2019-03-05-18-25-01-638_uk co lutraconsulting
screenshot_2019-03-05-18-27-44-386_uk co lutraconsulting

File: app/qml/FeaturePanel.qml [hunk 3]
```diff
@@ -175,8 +175,9 @@ Drawer {
                     property color backgroundColorInactive: "grey"
                     property color fontColor: InputStyle.fontColor
                     property color activeColor: InputStyle.fontColor
+                    property color attentionColor: "#aa0000"
                     property color normalColor: InputStyle.panelBackgroundLight
-                    property real cornerRadius: 1
+                    property real cornerRadius: 15 * QgsQuick.Utils.dp
```
Comment (wonder-sk) [inline app/qml/FeaturePanel.qml]: Umm.... I meant to turn 15 into `X * dp` while keeping the desired radius - `15*dp` is going to be too much :)

Comment (sklencar) [reply app/qml/FeaturePanel.qml]: I was thinking to decrease the value about the half, but actually now it seems fine to me. However, I am easy to have smaller radius.

**Pull request merged**